go.mod

```diff
@@ -38,21 +38,22 @@ require (
 	k8s.io/mount-utils v0.27.10
 	k8s.io/pod-security-admission v0.27.10
 	k8s.io/sample-controller v0.27.10
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
-	sigs.k8s.io/controller-runtime v0.17.0
+	k8s.io/utils v0.0.0-20230209194617-a36077c30491
+	sigs.k8s.io/controller-runtime v0.15.3
```

Comment:
we are going to an older version here?

Reply:
If you check the replace() section, we actually upgraded from 
`sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.14.5`
to 
`sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.15.3`